src/numbers/realnum.rs

```diff
@@ -63,8 +66,12 @@ impl RealNumber for f64 {
     }
 
     fn rand() -> f64 {
-        // TODO: to be implemented, see issue smartcore#214
-        1.0
+        let mut small_rng = SmallRng::from_seed(SEEDS);
```

Comment:
could you call https://github.com/smartcorelib/smartcore/blob/development/src/rand_custom.rs here?